src/test/java/io/debezium/connector/vitess/VitessValueConverterTest.java

```diff
@@ -61,11 +59,19 @@ public void before() {
         decoder = new VStreamOutputMessageDecoder(schema);
     }
 
+    @After
+    public void before() {
```

Comment:
`before` ro `@After`? I'd say the naming is a bit confusing :-)

Reply:
Nice catch, updated!